test/e2e_node/kubelet_test.go

```diff
@@ -44,12 +44,14 @@ var _ = Describe("Kubelet", func() {
 	})
 
 	Describe("pod scheduling", func() {
+		namespace := "pod-scheduling"
```

Comment:
Ah.  We are a ways off from being able to run multiple clusters on the same host in parallel so I would prefer to maximize simplicity at this point if you don't feel strongly about it.  Do the kubelet and apiserver log lines contain the namespace as well as the pod name?  I was hoping that by explicitly stating the namespaces per test that the logs would be easier to understand.